app/src/index.ts

```diff
@@ -114,16 +114,15 @@ function server() {
   proc.stderr.on('data', data => {
     logger.error(data.toString().trim())
   })
+    
 
-  function restart() {
-    logger.info('Restarting the server...')
-    server()
-  }
-
-  proc.on('exit', restart)
+  proc.on('exit', (code) => {
```

Comment:
Formatting should be `proc.on('exit', code =>` - can you run prettier against this file?